docs/raindex/example-strats/2-tranche-strat.md

```diff
@@ -0,0 +1,165 @@
+
+## Variants and control levers
+
+Almost everything about the strat is parameterized and chartable, e.g.
+* The algorithms that determine the price and amount of each tranche
+* The recharge rate and delay before the passive recharge starts to kick in
+* Whether the amounts per tranche are denominated in the input or output token
+* Whether the strategy is buying or selling TKN
+* An optional conversion between the input/output token and some external price, e.g. converting WETH in a vault to USD equivalent.
+
+You'll see these options in the strategy YAML frontmatter.
+
+### Growth curve
+The growth curves can govern the amount of tokens in each tranche as well as the price jump between each tranche. 
+
+These can be:
+- **Linear** - same differential between tranches
+- **Exponential** - differential grows each tranche. 
+
+### Start price
+Every tranche curve has a price for the first tranche, either:
+- The lowest price in a sell curve, where subsequent tranches will be at ever higher prices
+- The highest price in a buy curve, where subsequent tranches will be at ever lower prices
+
+### Start tranche
+Regardless of where the curve starts, when deploying the strategy you may decide to "initialise" at some tranche >1.
+
+Consider a sell curve where the price for the first tranche is $1, and each tranche increases linearly by 10c, e.g. the 5th tranche will be priced at $1.50.
+
+If the market is currently sitting at $1.57, when you deploy the strategy it's likely the first 5 tranches would immediately clear ($1.10, $1.20 etc), as all of those tranches are making an offer below the market and there is a clear arbitrage opportunity.
+
+To deal with this undesirable outcome we can initialise the strategy in the 6th tranche, or in other words, we can initialise the strategy as if tranches 1-5 have already cleared.
+
+### Recharge rate
+The recharge rate is defined by how many tranches we move through in a second. For the purposes of the rate, 1e18 is one tranche.
+
+For example, to recharge tranches once per day:
+
+```
+seconds in a day = 60 * 60 * 24 = 86400
+recharge rate = 1e18 / 86400 = 11574e9
+```
+
+What does this mean in practice? Consider the example above, where we have initialised our strategy in tranche 6, which means we have tokens available for $1.60. The market starts at $1.57, and for the first 24 hours it hovers at that value, edging up to just under $1.60.
+
+Tranches always recharge **in order** back to the first tranche, so immediately tranche 5 starts to recharge (ignore the delay for now, explained below).
+
+This means that over the course of the day, as that tranche recharges, there will be arbitrage opportunites and we'll see a full tranche of tokens clear at $1.50.
+
+### Recharge delay
+After a trade occurs, there is a delay before the next tranche starts recharging. This delay serves two primary purposes:
```

Comment:
the delay is on the _current_ tranche recharging

Reply:
fixed